doc/docusaurus/docs/using-plutus-tx/running-debugging.md

```diff
@@ -0,0 +1,328 @@
+  -O[INT]                             Set optimisation level; default: 0 , safe optimisations: 1, >=2: unsafe optimisations
+                 --whole-opt          Run an extra optimisation pass after all inputs are applied together. Ignored if only 1 input given.
+  -x SUFFIX                           Causes all files following this option on the command line to be processed as if they had the suffix SUFFIX
+  -n NAMING      --nam=NAMING         Change naming to `name` (default), `debruijn` or `named-debruijn`
+  -a ANNOTATION  --ann=ANNOTATION     Change annotation to `unit` (default) or `srcspan`
+                 --run                Compile and run
+                 --bench[=SECS]       Compile then run repeatedly up to these number of seconds (default:10) and print statistics
+                 --debug[=INTERFACE]  Compile then Debug program after compilation. Uses a `tui` (default) or a `cli` interface.
+                 --debug-dir[=DIR]    When `--debug`, try to search for PlutusTx source files in given DIR (default: .)
+                 --budget=INT,INT     Set CPU,MEM budget limit. The default is no limit. Only if --run, --bench, or --debug is given
+```
+Two interesting sub-commands which can aid during Plutus-based script development (e.g. Aiken, Plutus Tx, Plutarch) are `--print-builtins` and `--print-cost-model` which as their name suggest print detailed lists of currently recognized builtin functions and cost model parameters, respectively.
+
+## Compiling with the `plutus` CLI
+
+Before you can even run or debug your code, you first need to compile and extract the plutus code from the high-level source code.
+The steps for extracting plutus code vary depending on the source language you are using (Plutus Tx, Aiken, ...);
+if your are using Plutus Tx as source, you can follow the instructions on [how to extract compiled code].
+
+Although the CLI tool cannot help with compiling high-level code (e.g. Plutus Tx), it can aid with compiling and converting
+between different formats and intermediate representations of plutus code.
+
+In the following example we check a program written in the PIR (Plutus Intermediate Representation) language:
+
+``` shell
+$ echo "(program 1.1.0 (lam x (con integer) (lam y (con integer) x)))" > const.pir
+$ plutus const.pir
+Program passed all static checks. If you want the output result, use -o FILE or --stdout.
+```
+
+Or as a single step:
+
+``` shell
+$ echo "(program 1.1.0 (lam x (con integer) (lam y (con integer) x)))" | plutus -x pir --stdin
+Program passed all static checks. If you want the output result, use -o FILE or --stdout.
+```
+
+In the command above, the `-x pir` option is necessary for letting the CLI tool know
+the format/language of the supplied input. If the `-x` option is not passed, the CLI tool will try to guess the input's format/language
+by looking at its filename extension `.SUFFIX` --- and will default to `plc` for `--stdin` or no suffix.
```

Comment:
I believe `uplc` should be the default language, because it is what every compiler targets. Someone which writes Aiken code won't ever deal with `plc` or `pir`.

Reply:
Actually, is this a typo? It seems like you refer to typed plutus core as `tplc` in the rest of the document. So did you actually mean `uplc` here?